frontend/javascripts/oxalis/view/left-border-tabs/layer_settings_tab.tsx

```diff
@@ -1201,6 +1234,20 @@ class DatasetSettings extends React.PureComponent<DatasetSettingsProps, State> {
     });
   };
 
+  showAddSegmentIndexModal = (volumeTracing: VolumeTracing, layerName: string) => {
+    this.setState({
+      volumeTracingToAddSegmentIndex: volumeTracing,
+      volumeLayerNameToAddSegmentIndex: layerName,
```

Comment:
For volume annotations, the layer name is equal to `volumeTracing.tracingId` afaik. Therefore, you should only need one state variable instead of two.

Reply:
This is an example tracing id I printed to the console: `tracingId: "0aa32bb0-d1d6-4e34-b1f9-f88e1053c050`